source/nijigenerate/windows/settings.d

```diff
@@ -187,6 +189,24 @@ protected:
                             }
                         endSection();
                         break;
+                    case SettingsPane.Viewport:
+                        beginSection(__("Viewport"));
+                            string selected = incGetCurrentViewportZoomMode();
+                            if(igBeginCombo(__("Zoom Mode"), selected.toStringz)) {
+                                foreach (options; incGetViewportZoomModes()) {
```

Comment:
Should be two lines of setting code.
e.g.
if (igSelectable(__("To Screen Center"), incSettingsGet!string("ViewportZoomMode") == "ScreenCenter")) incSetViewportZoomMode("ScreenCenter");

Reply:
@seagetch The new commit has been pushed. Have you considered using a macro to replace the line? For long names, I think this might slightly reduce readability.